recipe/deploy/shared.php

```diff
@@ -27,10 +27,11 @@
         if (!test("[ -d $sharedPath/$dir ]")) {
             // Create shared dir if it does not exist.
             run("mkdir -p $sharedPath/$dir");
-            // If release contains shared dir, copy that dir from release to shared.
-            if (test("[ -d $(echo {{release_path}}/$dir) ]")) {
-                run("cp -r{$copyVerbosity} {{release_path}}/{$dir} {$sharedPath}/" . dirname(parse($dir)));
-            }
+        }
+
+        // If release contains shared dir, copy that dir from release to shared.
+        if (test("[ -d $(echo {{release_path}}/$dir) ]")) {
+            run("cp -ru{$copyVerbosity} {{release_path}}/{$dir} {$sharedPath}/" . dirname(parse($dir)));
```

Comment:
This is a really crucial part and needs to be tested.

Reply:
Sure, I still make the test.

Is there anything specific that I need to pay attention to in making the test?

Reply:
I think new test should cover use case of shared copy. Look at `joy` tests.